CHANGELOG.md

```diff
@@ -121,6 +121,7 @@
 
 ### Chore & Maintenance
 
+* `[docs]` Improve documentation of `mockClear`, `mockReset`, and `mockRestore` ([#6227](https://github.com/facebook/jest/pull/6227/files))
```

Comment:
Needs moved to the `master` section

Reply:
I think this got messed up after a merge conflict - cheers 😓 Moved back to `master` :)